rlp/src/stream.rs

```diff
@@ -344,6 +344,13 @@ impl RlpStream {
 		self.note_appended(1);
 		self.finished_list = true;
 	}
+
+	/// Finalize current unbounded list. Panics if no unbounded list has been opened.
+	#[deprecated(since = "0.4.3", reason = "use finalize_unbounded_list instead")]
```

Comment:
```suggestion
	#[deprecated(since = "0.4.3", note = "use finalize_unbounded_list instead")]
```
ah, sorry, it's actually called note, not reason

Reply:
Thanks!